packages/dds/tree/src/test/shared-tree/tableTree.bench.ts

```diff
@@ -32,10 +32,12 @@ import {
  * {@link runBenchmark} configuration.
  */
 interface BenchmarkConfig extends BenchmarkTimingOptions, TableBenchmarkOptions {
+	/* eslint-disable @fluid-internal/fluid/no-hyphen-after-jsdoc-tag -- false positive */
```

Comment:
Probably better to just remove this inheritDoc comment - this property just overrides a member of the same name on the base class.